scripts/_terraform_util.sh

```diff
@@ -148,6 +148,20 @@ _perform_terraform_action() {
         terraform --version
         terraform -chdir=/usr/primary init -input=false
         terraform -chdir=/usr/primary validate
+    elif [[ "${ACTION,,}" == "plan" ]]; then
+        terraform --version
+        if ! terraform -chdir=/usr/primary init -input=false >/dev/null; then
+            echo >&2 'terraform init failure'
+            exit 1
+        fi
+        if ! terraform -chdir=/usr/primary validate >/dev/null; then
```

Comment:
`terraform plan` should already cover failures from `terraform validate` no ? Do we need to run validate separately ?
We already have an action to run only `terraform validate`.